l5kit/l5kit/rasterization/rasterizer_builder.py

```diff
@@ -65,9 +73,27 @@ def build_rasterizer(cfg: dict, data_manager: DataManager) -> Rasterizer:
     history_num_frames = cfg["model_params"]["history_num_frames"]
 
     if map_type in ["py_satellite", "satellite_rgb"]:
```

Comment:
What's the difference between these two types? Do we need both?

Reply:
one is legacy, I'll remove it. good catch!